packages/design-system/src/components/Autocomplete/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -70,15 +70,15 @@ Array [
   <li
     aria-selected="false"
     class="ds-c-autocomplete__menu-item"
-    id="downshift-3-item-0"
+    id="downshift-4-item-0"
```

Comment:
this is interesting... looks like old id assigning behavior?

Reply:
Oh, hmm, at first I thought these ones were fine because they seemed to be based on the item index, but now I see that the root of it ("downshift-X") changed and is not deterministic.

Reply:
I'll try to fix this